.github/workflows/amend-user-repository-access.yml

```diff
@@ -17,7 +17,4 @@ jobs:
           architecture: "x64"
           cache: "pip"
       - run: python3 -m pip install -r scripts/python/requirements.txt
-      - run: python3 scripts/python/amend_user_repository_access.py ${{ secrets.OPERATIONS_ENGINEERING }} > output.txt
```

Comment:
This currently prevents the output from being visible in the GitHub action. 

We don't use this in any other workflow and we aren't using this output anywhere.

Removing it so it's easier to debug the logs whilst the job is running and see if there are any silent issues not currently being reported.

Reply:
It was added so could read the output in the file. Maybe had a plan it would be read by another script for metrics. Can be removed if need be.

Reply:
Will leave for now as there was concerns about personal data (ticket for that [here](https://app.zenhub.com/workspaces/operations-engineering-614da9b850253700151e7252/issues/gh/ministryofjustice/operations-engineering/2147)